modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java

```diff
@@ -137,9 +144,11 @@ private void _checkDDMFormFieldParameterNames(
 				ddmFormFieldParameterNames.add(
 					defaultDDMFormFieldParameterName);
 
+				fullDDMFormFields.put(ddmFormField.getName(), ddmFormField);
+
```

Comment:
Don't you think that we could only add the fields once? I would leave the code on lines 163-164 because I don't see the difference or advantage of adding before. What is your opinion about it?